cli/valet.php

```diff
@@ -508,14 +508,25 @@
         if (! $phpVersion) {
             $path = getcwd().'/.valetphprc';
             $linkedVersion = Brew::linkedPhp();
-            if (! file_exists($path)) {
-                return info("Valet is using {$linkedVersion}.");
+
+            if (file_exists($path)) {
+                $phpVersion = trim(file_get_contents($path));
+                info("Found '{$path}' specifying version: {$phpVersion}");
+            }
+
+            if (! $phpVersion) {
+                $site = basename(getcwd()).'.'.data_get(Configuration::read(), 'tld');
+                if ($phpVersion = Site::customPhpVersion($site)) {
+                    $phpVersion = PhpFpm::normalizePhpVersion($phpVersion);
+                    info("Found isolated site '{$site}' specifying version: {$phpVersion}");
+                }
             }
 
-            $phpVersion = trim(file_get_contents($path));
-            info("Found '{$path}' specifying version: {$phpVersion}");
+            if (! $phpVersion) {
+                return info("Valet is using {$linkedVersion}.");
+            }
 
-            if ($linkedVersion == $phpVersion) {
+            if ($linkedVersion == $phpVersion && ! $force) {
```

Comment:
Previously, it wasn't respecting `--force` when comparing version against `.valetphprc`. I thought it would do that.